internal/service/ecs/cluster.go

```diff
@@ -247,26 +250,26 @@ func resourceClusterRead(d *schema.ResourceData, meta interface{}) error {
 	defaultTagsConfig := meta.(*conns.AWSClient).DefaultTagsConfig
 	ignoreTagsConfig := meta.(*conns.AWSClient).IgnoreTagsConfig
 
-	var out *ecs.DescribeClustersOutput
+	var cluster *ecs.Cluster
 	err := resource.Retry(2*time.Minute, func() *resource.RetryError {
```

Comment:
```suggestion
	err := resource.Retry(clusterReadTimeout, func() *resource.RetryError {
```
While you're here, constants are better.